### PR TITLE
fix: change fira source

### PR DESCRIPTION
**Pull request by wusteven815**
- Adds #1902
- Use `@fontsource` instead of GitHub as source
- Lock to Fira Sans to 5.0.20 and Fira Mono to 5.0.13 to prevent kerning problems

### CODE REVIEW

**Comment by mofojed** [comment]
@dsmmcken would these differences be expected/okay? Fonts appear okay to me, tabular numbering etc.

**Actual:**
typography-actual

**Expected:**
typography-expected

**Diff:**
typography-diff


From in the app on a table with numbers:
image

**Review by dsmmcken — DISMISSED** [review]
Missing `Fira Mono`, which is being substituted. The previous package included both Fira Sans and Mono, you'll have to add those in as well.

That's frustrating that the kerning tables must be different. Still looks acceptable, and has correct font-features-settings. 🤷

**Comment by dsmmcken** [comment]
null

We are probably also missing an italic version in that weight.

File: packages/app-utils/src/components/FontBootstrap.tsx [hunk 1]
```diff
@@ -1,5 +1,11 @@
 import React, { createContext, useEffect, useState } from 'react';
-import 'fira';
+import '@fontsource/fira-mono/400.css';
+import '@fontsource/fira-sans/300.css';
+import '@fontsource/fira-sans/400.css';
+import '@fontsource/fira-sans/400-italic.css';
+import '@fontsource/fira-sans/500.css';
+import '@fontsource/fira-sans/600.css';
+import '@fontsource/fira-sans/700.css';
```
Comment (dsmmcken) [inline packages/app-utils/src/components/FontBootstrap.tsx:8]:
```suggestion
import '@fontsource/fira-sans/300.css';
import '@fontsource/fira-sans/300-italic.css';
import '@fontsource/fira-sans/400.css';
import '@fontsource/fira-sans/400-italic.css';
import '@fontsource/fira-sans/500.css';
import '@fontsource/fira-sans/600.css';
import '@fontsource/fira-sans/700.css';
import '@fontsource/fira-sans/700-italic.css';
```

There's a couple places with 300 italic and 700 italic

**Review by dsmmcken — APPROVED** [review]

**Review by mofojed — APPROVED** [review]

**Review by mofojed — CHANGES_REQUESTED** [review]
Need to re-run `npm install` to update the `package-lock.json` now.

**Review by dsmmcken — APPROVED** [review]

**Pull request merged**